Content.Server/Chat/Commands/NukeChatMessagesCommand.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Content.Server.Administration;
+using Content.Server.Chat.V2.Repository;
+using Content.Shared.Administration;
+using Robust.Shared.Console;
+
+namespace Content.Server.Chat.Commands
+{
+    [AdminCommand(AdminFlags.Admin)]
+    internal sealed class NukeChatMessagesCommand : IConsoleCommand
+    {
+        public string Command => "nuke";
+        public string Description => "Delete all of a user's specific chat message";
+        public string Help => "nuke <username>";
+
+        public void Execute(IConsoleShell shell, string argStr, string[] args)
+        {
+            IoCManager.Resolve<IEntityManager>().System<ChatRepository>().Nuke(args[0]);
```

Comment:
It's probably not for the best to validate user here as it would assume an otherwised purged or absent user would appear in whatever query I make.

A string is safe as if a user called joeBloggins ever posts that string being nuked gets their messages, no infra required.